package.json

```diff
@@ -150,6 +150,13 @@
           ".yaml"
         ]
       }
+    ],
+    "commands": [
+      {
+        "command": "jbang.apache.camel.routes",
```

Comment:
for the ID, we should start with camel given that we are in the Camel extension. , it is a bit like a namespace. here we can think that the command is provided by the VS Code JBang extension.

Reply:
that make sense